Miscellaneous style guide edits (#6441)
braze-inc/braze-docs: _docs/_api/endpoints/user_data/post_user_track.md

Consider the following use cases where you may use the bulk update header:

- A daily job where multiple users' custom attributes are updated via the `/users/track` endpoint.

> - An impromptu user data backfill script which updates user information via the `/users/track` endpoint.

Author: ```suggestion
- A user data backfill script that updates user information via the `/users/track` endpoint.
```
Confirmed with Dave from Ingestion
